### PR TITLE
add batch mode for neutron activator

### PR DESCRIPTION
**Pull request by GlodBlock**

### CODE REVIEW

**Comment by Dream-Master** [comment]
Fix or feature ? Ask because 2.3. need to be done soon.

**Comment by GlodBlock** [comment]
feature, no hurry to be merged

**Comment by Zereff06** [comment]
https://github.com/GTNewHorizons/GoodGenerator/blob/8dfa17fbeb62675504bf0fc39103a4e9bc458db6/src/main/java/goodgenerator/blocks/tileEntity/NeutronActivator.java#L135

if (lastRecipe == null) then NullPointerException ?

**Comment by GlodBlock** [comment]
> https://github.com/GTNewHorizons/GoodGenerator/blob/8dfa17fbeb62675504bf0fc39103a4e9bc458db6/src/main/java/goodgenerator/blocks/tileEntity/NeutronActivator.java#L154
> 
> if (lastRecipe == null) then NullPointerException ?

it is checked in L129

File: src/main/java/goodgenerator/blocks/tileEntity/NeutronActivator.java [hunk 1]
```diff
@@ -153,6 +190,7 @@ public void loadNBTData(NBTTagCompound aNBT) {
         mCeil = aNBT.getInteger("mCeil");
         mFloor = aNBT.getInteger("mFloor");
         height = aNBT.getInteger("height");
+        mBatchMode = aNBT.getBoolean("mBatchMode");
```
Comment (minecraft7771) [inline src/main/java/goodgenerator/blocks/tileEntity/NeutronActivator.java:193]: Please use the variables that are now in the multiblock class. `batchMode` and `BATCH_MODE_NBT_KEY`. Makes future GUI transitions easier.

**Review by minecraft7771 — CHANGES_REQUESTED** [review]

File: src/main/java/goodgenerator/blocks/tileEntity/NeutronActivator.java (hunk 1 above)
Comment (GlodBlock) [reply src/main/java/goodgenerator/blocks/tileEntity/NeutronActivator.java]: done

**Comment by minecraft7771** [comment]
Only dismissed for now so dream doesn't feel the urge to merge :trollface:

**Review by minecraft7771 — APPROVED** [review]

**Pull request merged**